docs/spec/broker.md

```diff
@@ -40,10 +40,28 @@ default Broker upon creation if no Broker is specified by the user.
 A Trigger MAY be created before its assigned Broker exists. A Trigger SHOULD
 progress to Ready when its assigned Broker exists and is Ready.
 
+#### Event filters
+
+Implementations of Trigger MUST support both attributes filter and expression
```

Comment:
what happens when both are given?

Reply:
I wrote that at the end of this paragraph, the trigger MUST do an and of both results. Do you propose to rephrase it?